packages/map-template/src/components/MapTemplate/MapTemplate.scss

```diff
@@ -47,6 +47,46 @@
     .mi-floor-selector__list {
         box-sizing: unset;
     }
+
+    .gmnoprint {
+        background-color: red;
```

Comment:
Is this intended?

![image](https://github.com/MapsPeople/web-ui/assets/50314165/bbaa15f1-e7fb-4ec8-86d6-020f1bdf1b98)

Reply:
Removed.